ads_plugin.py

```diff
@@ -434,14 +429,15 @@ def on_save_segmentation_button(self, event):
 
         # Save the arrays as PNG files
         myelin_and_axon_array = (myelin_array // 2 + axon_array).astype(np.uint8)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg.png", img=myelin_and_axon_array)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg-myelin.png", img=myelin_array)
-        ads_utils.imwrite(filename=save_dir + "/ADS_seg-axon.png", img=axon_array)
+        ads_utils.imwrite(filename=(save_dir / "ADS_seg.png"), img=myelin_and_axon_array)
```

Comment:
Either remove the brackets in the filename argument or keep the brackets for myelin and axon mask as well .

Reply:
Done!